### PR TITLE
Start documenting keyboard shortcuts

### PR DESCRIPTION
**Pull request by ptman**
I'm sure I don't know all of them.

### CODE REVIEW

File: docs/shortcuts.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Keyboard Shortcuts
+
+- <kbd>Ctrl</kbd>+<kbd>m</kbd> - toggle markdown
```
Comment (t3chguy) [inline docs/shortcuts.md]: these are actually Control or Command depending on whether you're not on a Mac

Comment (ptman) [reply docs/shortcuts.md]: all of them or just this one?

Comment (ptman) [reply docs/shortcuts.md]: Better?

**Comment by aaronraimist** [comment]
Looks good to me

@t3chguy?

**Comment by jryans** [comment]
Thanks for making this contribution a while back. It looks like this one feel through the cracks a bit. I have added it to the review queue, so hopefully we'll get a fresh look at it soon.

**Comment by ptman** [comment]
@jryans no worries, I get it that you have a huge load of issues and PRs

**Comment by jryans** [comment]
I think it would be fine to land this as-is. We can always update or extend it later.

@ptman Please update the PR or each commit with your sign off as described in [CONTRIBUTING.rst](https://github.com/matrix-org/matrix-js-sdk/blob/develop/CONTRIBUTING.rst). We should be able to merge this once that's done.

**Comment by jryans** [comment]
I am assuming the test failures in CI are unrelated since there are no code changes here.

**Comment by ptman** [comment]
Yes, I certainly hope there are more, but those were the only ones I could find when I wrote this. Would especially like a shortcut to switch to the next unread room.

**Pull request merged**